### PR TITLE
Restructuring to get Chris' branch working

### PR DESCRIPTION
**Pull request by jsignell**
* Took out command line tool
 * Restructuring docs to incorporate content from README
 * Moving dependencies from setup.cfg to setup.py
 * Adding some unit tests

### CODE REVIEW

**Comment by jsignell** [comment]
This superseded #16

**Comment by jsignell** [comment]
Ok this passed CI finally. @jbednar, if you approve I'll merge it.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            all_branches: true
+
+    - <<: *website
+      stage: website_dev
+      deploy:
+        - provider: pages
+          skip_cleanup: true
+          github_token: $GITHUB_TOKEN
+          local_dir: ./builtdocs
+          repo: ioam-docs/colorcet-dev
```
Comment (jbednar) [inline .travis.yml]: pyviz-dev?

File: colorcet/__init__.py [hunk 2]
```diff
@@ -13820,3 +13821,18 @@ def mpl_cm(name,colorlist):
 
 palette_n = AttrODict(sorted(palette_n.items()))
 cm_n = AttrODict(sorted(cm_n.items()))
+
+
+# make pyct's example/data commands available if possible
```
Comment (jbednar) [inline colorcet/__init__.py]: examples can be removed.

File: doc/conf.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+from nbsite.shared_conf import *
+
+project = 'colorcet'
+authors = u'Anaconda'
+copyright = u'2019 ' + authors
```
Comment (jbednar) [inline doc/conf.py]:
```suggestion
copyright = u'2017-2019 ' + authors
```

File: dodo.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import os
```
Comment (jbednar) [inline dodo.py]: Can you add a comment to the top line of each dodo.py file pointing to how it's used?

File: conda.recipe/meta.yaml [hunk 5]
```diff
@@ -1,21 +1,33 @@
+{% set sdata = load_setup_py_data() %}
```
Comment (jsignell) [inline conda.recipe/meta.yaml]: get rid of this template file

**Comment by jsignell** [comment]
Ok I think this is ready to go in @jbednar. I don't have permission, so you can do it or I can once you give it to me :)

**Comment by jsignell** [comment]
screen shot 2019-02-13 at 1 58 04 pm

**Pull request merged**